src/pages/_includes/graphql/catalog-service/product-view.md

```diff
@@ -79,6 +81,15 @@ Field | Data Type | Description
 `roles` | [String] | A list that describes how the image is used. Can be `image`, `small_image`, or `thumbnail`.
 `url` | String! | The URL to the product image.
 
+### ProductViewLink type
+
+The `ProductViewLink` type contains details about product links for related products and cross selling.
+
+Field | Data Type | Description
+--- | --- | ---
+`product` | ProductView! | Details about the product in the link.
+`linkTypes` | [String!]! | Types of links for this product. Can be `crossell`, `related`, and `upsell`.
```

Comment:
Double check that `crossell` doesn't have 3 consecutive esses, as it does in core.

Reply:
Good catch.